complex/ft_config.c

```diff
@@ -317,11 +324,13 @@ static int ft_parse_key_val(char *config, jsmntok_t *token, char *test_set)
 					val_token[i].end - val_token[i].start);
 			break;
 		case VAL_NUM:
-			ft_parse_num(config + val_token[i].start, key,
-					test_set + key->offset + key->val_size * i);
+			if (ft_parse_num(config + val_token[i].start, key,
+					test_set + key->offset + key->val_size * i) < 0)
+				return -1;
```

Comment:
I'm going to go ahead and merge this, but it would be better to return the value returned from ft_parse_num(), rather than checking it and returning -1 directly.